server/README.md

```diff
@@ -28,26 +28,13 @@ Navigate to `http://localhost:3002/api-docs` to view the Swagger server API refe
 ## Testing
 The project utilizes [Mocha](https://mochajs.org/) and [Chai](https://www.chaijs.com/) to run backend integration tests.
 
-Copy the env.TEMPLATE file to .env and set this variable:
-```
-NODE_ENV=test
-```
-
 From the website directory, run:
 ```
-docker compose -f docker-compose.yml -f docker-compose.test.yml up -d
+docker compose -f docker-compose.yml -f docker-compose.test.yml -p test up -d
```

Comment:
Should the workflow also be updated to run this command? What does `-p test` do?

Reply:
-p test gives a project label to the containers. This gives them a unique name so that the test containers and the dev containers won't overwrite one another.
![image](https://user-images.githubusercontent.com/49756432/146560141-53fcad62-efb1-4980-9ac1-8132988dc3a6.png)

This is only necessary in dev as the workflow won't be running the dev containers, only the test containers.